bin/metrics-billing.rb

```diff
@@ -35,7 +35,7 @@ class BillingMetrics < Sensu::Plugin::Metric::CLI::Graphite
   include Common
 
   option :services_name,
-         short: '-s SERVICES_NAME',
+         short: '-S SERVICES_NAME',
```

Comment:
@eheydrick we have seen lots of these types of issues across a bunch of plugins. I think we should discuss possibly only specifying short names for options that are standard like `-w, -c, -h, -p` what are your thoughts?